### PR TITLE
add support for extra class in menu links

### PR DESCRIPTION
**Pull request by tobiasehlert**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Adds possibility to add extra class(es) to navigation-link items in the menu.

### Issues Resolved

_n/a_

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [ ] Explain why and how the changes were necessary and implemented respectively
- [ ] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [ ] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [ ] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by tobiasehlert** [comment]
Would maybe even be better if we also add support for "active" page support as in example:
https://gohugo.io/templates/menu-templates/#example

**Comment by luizdepra** [comment]
Sorry for taking so long to reply here. Well...

Sure. We could use this example as a base to add multi-level menus.

If you're interested, feel free to send a PR changing this code.

File: layouts/partials/header.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
         {{ with .Site.Menus.main}}
           {{ range sort . }}
             <li class="navigation-item">
-              <a class="navigation-link" href="{{ index (apply (slice .URL) (.Params.urlFunc | default "relLangURL") ".") 0 }}">{{ .Name }}</a>
+              <a class="navigation-link {{ .Params.class }}" href="{{ index (apply (slice .URL) (.Params.urlFunc | default "relLangURL") ".") 0 }}">{{ .Name }}</a>
```
Comment (luizdepra) [inline layouts/partials/header.html:15]: Let me just ask for one more thing. Could you add an entry in the `docs` folder about this new configuration parameter?

Comment (tobiasehlert) [reply layouts/partials/header.html:15]: @luizdepra, I've added a entry in the docs with commit 28d4538.
Is that good enough or you want me to extend that even further?

Comment (luizdepra) [reply layouts/partials/header.html:15]: LGTM. Ty. :)

**Pull request merged**